packages/example-getting-started/README.md

```diff
@@ -0,0 +1,419 @@
+# @loopback/example-getting-started
+
+This is the basic tutorial for getting started with Loopback 4!
+
+**NOTICE**: This tutorial is currently under construction! This notice will be
+removed when it is ready for use!
+
+## Prerequisites
+
+Before we can begin, you'll need to make sure you have some things installed:
+- [Node.js](https://nodejs.org/en/) at v6.x or greater
+- [TypeScript](http://www.typescriptlang.org/)
```

Comment:
A global installation of TypeScript should not be needed, typescript should be a dev-dependency of this package.

I know this is probably out of scope of this PR, but since we have to update the instructions anyways (see below), then I think it's best to fix this part too.